packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -149,6 +150,92 @@ export class ProductCard extends PureComponent {
         );
     }
 
+    renderBundlePriceBadge(bounds) {
+        const {
+            product: { type_id, price_view }
+        } = this.props;
+
+        if (type_id !== BUNDLE) {
+            return null;
+        }
+
+        // eslint-disable-next-line
+        let bundleBadgeLabel;
+
+        if (bounds === 'bottom') {
+            bundleBadgeLabel = 'From';
```

Comment:
Please wrap the translatable strings in the translation function `__()`.
Do the same for the other strings in your code.
https://docs.create-scandipwa-app.com/building-your-app/internationalization#writing-translatable-code